content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -310,8 +310,7 @@ kubectl logs -f my-pod                              # stream pod logs (stdout)
 kubectl logs -f my-pod -c my-container              # stream pod container logs (stdout, multi-container case)
 kubectl logs -f -l name=myLabel --all-containers    # stream all pods logs with label name=myLabel (stdout)
 kubectl run -i --tty busybox --image=busybox -- sh  # Run pod as interactive shell
-kubectl run nginx --image=nginx -n 
-mynamespace                                         # Run pod nginx in a specific namespace
+kubectl run nginx --image=nginx -n mynamespace      # Start a single instance of nginx pod in the namespace of mynamespace
```

Comment:
Just a nit.
Although correct, how about
```suggestion
kubectl run nginx --image=nginx -n mynamespace      # Run pod nginx in the "mynamespace" namespace
```
for a bit of parallelism in the examples